src/app/setup.py

```diff
@@ -28,19 +28,21 @@ def find_version(version_file):
     license="MIT",
     keywords="beer beer-garden beergarden",
     install_requires=[
-        "apispec==0.38.0",
-        "apscheduler==3.6.3",
+        "apispec<0.39",
+        "apscheduler<4",
         "brewtils>=3.0.2",
+        "marshmallow<3",
         "mongoengine<0.21",
         "more-itertools<9",
         "motor<3",
         "passlib<1.8",
-        "prometheus-client==0.7.1",
-        "pyrabbit2==1.0.7",
+        "prometheus-client<1",
+        "pyyaml<6",
+        "pyrabbit2<2",
         "pytz<2021",
         "ruamel.yaml<0.17",
         "stomp.py<6.2.0",
-        "tornado==6.0.4",
+        "tornado<7",
         "yapconf>=0.3.7",
```

Comment:
I think you based this off an old `requirements.txt`. https://github.com/beer-garden/beer-garden/pull/1167 discusses that the `wheel` package needs to be explicitly installed in Python 3.7+.